src/routes/github/subscription/github-subscription-get.ts

```diff
@@ -4,18 +4,23 @@ import { isUserAdminOfOrganization } from "utils/github-utils";
 import { createAppClient, createUserClient } from "~/src/util/get-github-client-config";
 
 export const GithubSubscriptionGet = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
-	const { githubToken, jiraHost, gitHubAppId } = res.locals;
-
+	const { githubToken, jiraHost } = res.locals;
 	const gitHubInstallationId = Number(req.params.installationId);
+	const gitHubAppId = Number(req.query.gitHubAppId);
```

Comment:
Same security concern.

Reply:
Same thing here. Unless someone knows what my GH token is, as well as my jiraHost, I don't see any issues with passing the gitHubAppId from the query string.